pizza_cutter/des_pizza_cutter/_pizza_cutter.py

```diff
@@ -230,6 +246,12 @@ def _make_epochs_info(object_data, weights, all_se_info):
         data['fname'][i] = se_info['filename']
         data['file_id'][i] = se_info['file_id']
 
+        sl = all_slices[i]
+        data['row_start'][i] = sl.y_start
+        data['col_start'][i] = sl.x_start
+        data['box_size'][i] = sl.box_size
+        data['psf_box_size'][i] = sl.psf_box_size
```

Comment:
We can set it here using the attributes `psf_x_start` and `psf_y_start` attached to the slice object.

Reply:
why do we care about that?

Reply:
Determines the WCS of each pixel in the input images to the sum.